windowslib/src/main/scala/scala/scalanative/windows/util/Conversion.scala

```diff
@@ -13,4 +14,19 @@ private[windows] object Conversion {
   def wordFromBytes(low: Byte, high: Byte): WinWord = {
     ((low & 0xff) | ((high & 0xff) << 8)).toUShort
   }
+
+  def dwordPairToULargeInteger(high: DWord, low: DWord): ULargeInteger = {
+    if (high == 0.toUInt) low
+    else (high.toULong << 32) | low
+  }
+
+  def uLargeIntegerToDWordPair(
+      v: ULargeInteger,
+      high: Ptr[DWord],
+      low: Ptr[DWord]
+  ): Unit = {
+    val mask = 0xffffffff.toUInt
+    !high = ((v >> 32) & mask).toUInt
+    !low = (v & mask).toUInt
```

Comment:
`& mask` is redundant with `.toUInt`:
```suggestion
    !high = (v >> 32).toUInt
    !low = v.toUInt
```